README.md

```diff
@@ -1,14 +1,35 @@
-# Project
-
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
-
-As the maintainer of this project, please make a few updates:
-
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+# Python environment tools for Visual Studio Code
+
+Performant Python environment tooling and support, such as locating all global Python installs and virtual environments.
+
+## Environment Types Supported 
+
+- python.org
+- Windows Store
+- PyEnv
+- PyEnv-Win
+- PyEnv-Virtualenv
+- Conda
+- Miniconda
+- Miniforge
+- PipEnv
+- Homebrew
+- VirtualEnvWrapper
+- VirtualEnvWrapper-Win
+- Venv
+- VirtualEnv
+- Python on your PATH
+- Mamba
+- Micromamba
```

Comment:
Lets leave this out for now, its not easy to get the exe for micromamba,
We can add this later.
We might end up discovering the envrionments, but perhaps not the conda exe.

Reply:
Please remove this line